core-services/release-controller/_releases/priv/release-ocp-4.13.json

```diff
@@ -189,6 +189,14 @@
                 "name": "periodic-ci-openshift-release-master-ci-4.13-e2e-azure-sdn-techpreview-serial-priv"
             }
         },
+        "azure-sdn-upgrade-micro": {
+            "disabled": true,
+            "optional": true,
+            "prowJob": {
+                "name": "periodic-ci-openshift-release-master-ci-4.14-e2e-azure-sdn-upgrade-priv"
+            },
+            "upgrade": true
+        },
```

Comment:
This looks like it might be in the wrong file, was this hand edited? We've got a 4.14 job on a 4.13 release file. Looks like it just needs to go to a diff one.